frontend/src/router/index.js

```diff
@@ -101,6 +101,21 @@ const Message = () => import('@/views/apps/email/Message')
 
 Vue.use(Router)
 
+const validPathNames = [
```

Comment:
This change was most likely carried over from #42 by accident. This has been resolved by @purplesmurf1998 and myself on the previous PR.

Reply:
Already addressed this in Qandeel's reply. Thanks for reviewing the code.